birdhouse/config/thredds/config/magpie/providers.cfg.template

```diff
@@ -0,0 +1,35 @@
+providers:
+  thredds:
+    url: http://thredds:8080/twitcher/ows/proxy/thredds
```

Comment:
Same as per other comment.

Reply:
This is not included in #331 since this is being moved from the `catalog` component.

If you'd like I can make this use the external port 8083 and then change it again when/if #331 gets pulled in but it seems like we might as well do it right the first time (in this PR) since this is new here.